codegen/tests/cardspb/cards_api.pb.go

```diff
@@ -6,12 +6,11 @@ package cardspb
 import (
 	context "context"
 	fmt "fmt"
-	math "math"
-
 	proto "github.com/golang/protobuf/proto"
 	grpc "google.golang.org/grpc"
 	codes "google.golang.org/grpc/codes"
 	status "google.golang.org/grpc/status"
+	math "math"
```

Comment:
This file, `codegen/tests/cardspb/wallet_api.pb.go` and `codegen/tests/simplepb/simplegrpc.pb.go` shouldn't be changed after `goimports -w .` run. Might related to Makefile?

Reply:
Will fix. Thanks.